src/main/bash/sdkman-env.sh

```diff
@@ -30,14 +30,26 @@ function __sdk_env() {
 	local line_number=0
 
 	while IFS= read -r line || [[ -n $line ]]; do
-		if [[ ! $line =~ ^[[:lower:]]+=.+$ ]]; then
-			__sdkman_echo_red "${sdkmanrc}:${line_number}: Invalid candidate format! Expected 'candidate=version' but found '$line'"
+		__sdkman_is_blank_or_comment "$line" && continue
+
+		if ! __sdkman_matches_candidate_format "$line"; then
+			__sdkman_echo_red "${sdkrc}:${line_number}: Invalid candidate format! Expected 'candidate version' but found '$line'"
```

Comment:
Can we split this into two lines please, separated by a newline .
The warning in red, the further information in yellow.